service/src/registry.ts

```diff
@@ -556,6 +638,443 @@ export class Registry {
     this._restApiClient.executeApiCall('GET', path, {}, null, done);
   }
 
+  /**
+   * @method            module:azure-iothub.Registry#addConfiguration
+   * @description       Add a configuration to an IoT hub.
+   *
+   * @param {Configuration} configuration An object of type module:azure-iothub.Configuration
+   *                                      to add to the hub
+   * @param {Function}      done          The function to call when the operation is
```

Comment:
>done [](start = 28, length = 4)

Bla bla not required bla bla third parameter bla bla here and other places but not get